lib/pbench/test/unit/server/conftest.py

```diff
@@ -83,12 +82,22 @@ def teardown():
 
 
 @pytest.fixture
-def client(pytestconfig, monkeypatch):
+def server_config(pytestconfig, monkeypatch):
     cfg_file = pytestconfig.cache.get("_PBENCH_SERVER_CONFIG", None)
     monkeypatch.setenv("_PBENCH_SERVER_CONFIG", cfg_file)
-    app = create_app()
-    app.testing = True
+
+    server_config = get_server_config()
+    return server_config
+
+
+@pytest.fixture
+def client(pytestconfig, monkeypatch, server_config):
```

Comment:
Since you refactored the only `monkeypatch` reference into the `server_config` fixture, I don't think you need `monkeypatch` here anymore. Nor probably `pytestconfig` for that matter...

Reply:
yes you are right its redundant